celery_worker/worker.py

```
@@ -74,41 +72,39 @@ def process_drawing(self, drawing_id: str, question: str):
             answer = None
             error_msg = result.get("error", "Unknown agent error")
 
-    except requests.exceptions.HTTPError as http_err:
-        # Ретрай при временных ошибках сети/агента
-        if response.status_code in (502, 503, 504):
-            logger.warning(f"Ошибка {response.status_code}, пробую еще раз...")
-            raise self.retry(exc=http_err, countdown=30)
-        status = "failed"
-        error_msg = f"HTTP {response.status_code}: {response.text}"
-        answer = None
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)